src/schema/schema.graphql

```diff
@@ -2580,7 +2580,6 @@ input ExternalAccountModifyAttrsInput {
 	accountType: AccountType
 	defaultSignature: ID
 	description: String
-	emailAddress: String
```

Comment:
upon sending emailAddress to backend it was throwing the "Data source already exists" error
so removed this

Reply:
will this change have any impact on add new account?

Reply:
nope we are modifying "ExternalAccountModifyAttrsInput" only